### PR TITLE
[winlog] Add a couple ECS fields for error checking/codes

### PR DESCRIPTION
**Pull request by nicpenning**
- Bug

## What does this PR do?

Account for any ingest pipelines that might throw some error messages or event codes.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/winlog/docs/README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ To achieve this, `renderXml` needs to be set to `1` in your [inputs.conf](https:
 | data_stream.namespace | Data stream namespace. | constant_keyword |
 | data_stream.type | Data stream type. | constant_keyword |
 | ecs.version | ECS version this event conforms to. `ecs.version` is a required field and must exist in all events. When querying across multiple indices -- which may conform to slightly different ECS versions -- this field lets integrations adjust to the schema version of the events. | keyword |
+| error.code | Error code describing the error. | keyword |
+| error.message | Error message. | match_only_text |
```
Comment (efd6) [inline packages/winlog/docs/README.md:47]: Please update this via the _dev/build/docs/README.md.

Comment (nicpenning) [reply packages/winlog/docs/README.md:47]: I would be happy to!

What is syntax for that?

{{fields "winlog", "ecs", "..."}}

Or is it something more complex than that?

Trying to follow other integrations, it doesn't quite make sense to me on how these are being built.

**Comment by nicpenning** [comment]
Had a couple lingering spaces :/ 

Need to test again

File: packages/winlog/docs/README.md (hunk 1 above)
Comment (efd6) [reply packages/winlog/docs/README.md:47]: Sorry, my error.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**